README.md

```diff
@@ -2,3 +2,4 @@
 Native implementation for the [`ernnavigation-api`](https://github.com/electrode-io/ern-navigation-api#electrode-native-navigation-api)
 
 
+Please run `./ios/createContainer.sh` to get the iOS Container, and run `/ios/moviesreloadedMiniApp/ErnRunner.xcodeproj` from XCode to try Electrode Native Navigation on iOS side.
```

Comment:
_open_ (not run) <xocdeproj> in _Xcode_ (not XCode). Also instead of _two_ blank lines before this line, only use one.